shared/common-adapters/list-item.desktop.js

```diff
@@ -6,9 +6,9 @@ import type {Props} from './list-item'
 
 export default class ListItem extends Component<void, Props, void> {
   render () {
-    const clickable = this.props.clickable === undefined ? true : !!this.props.clickable
+    const clickable = !!this.props.onClick
```

Comment:
i dunno if we want to do this. the item itself can have no onclick handler but things in it might